data/json/items/armor/torso_clothes.json

```diff
@@ -939,6 +939,75 @@
     "color": "white",
     "warmth": 5,
     "material_thickness": 0.1,
+    "variant_type": "generic",
+    "variants": [
+      {
+        "id": "skeletor_tshirt",
+        "name": { "str_sp": "political meme t-shirt" },
```

Comment:
`str_sp` means the plural form is the same as the singular form and I don't think it should be used here.

Reply:
Thanks!